src/Tribe/Aggregator/Record/List_Table.php

```diff
@@ -497,14 +497,20 @@ public function column_total( $post ) {
 			// always show created
 			$created = $last_imported->get_event_count( 'created' );
 
-			$html[] = esc_html__( 'New: ', 'the-events-calendar' ) . ( $created ? $created : 0 );
-
-			if ( $updated = $last_imported->get_event_count( 'updated' ) ) {
-				$html[] = esc_html__( 'Updated: ', 'the-events-calendar' ) . $updated;
-			}
+			$html[] = esc_html__( 'Latest Import: ', 'the-events-calendar' ) . ( $created ? $created : 0 );
 		}
 
 		if ( 'schedule' === $record->type ) {
+			if ( ! empty( $record->post->post_parent ) ) {
+				$created = $record->get_event_count( 'created' );
+
+				$html[] = esc_html__( 'New: ', 'the-events-calendar' ) . ( $created ? $created : 0 );
```

Comment:
Unless, I'm misreading this, isn't this New that same as what will show for Total Events below?